package.json

```diff
@@ -1,29 +1,82 @@
 {
-  "name": "my-project",
-  "version": "0.1.0",
+  "name":"WeeUrl",
+    "version": "1.0.2",
+    "description": "Reliable Development Service At Scale",
+    "keywords": [
+        "typescript",
+        "react",
+        "next",
+        "eslint",
+        "prettier",
+        "jest",
+        "google",
+        "google-analytics"
+    ],
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "build": " next build",
+    "start": "next start"
   },
   "dependencies": {
-    "@radix-ui/react-popover": "1.0.2",
-    "@types/node": "18.11.10",
-    "@types/react": "18.0.26",
-    "@types/react-dom": "18.0.9",
-    "eslint": "8.29.0",
-    "eslint-config-next": "13.0.6",
-    "next": "13.0.6",
+    "@emailjs/browser": "^3.10.0",
+    "@fontsource/karla": "^4.5.10",
+    "@fontsource/sora": "^4.5.10",
+    "@headlessui/react": "^1.7.4",
+    "@heroicons/react": "^2.0.13",
+    "@radix-ui/react-accordion": "^1.0.1",
```

Comment:
I think we should decide whether we'll use `radix-ui` or tailwind. Using both is inappropriate IMO.